deepmd/pt/model/model/dp_model.py

```diff
@@ -95,30 +23,3 @@ def update_sel(cls, global_jdata: dict, local_jdata: dict):
             global_jdata, local_jdata["descriptor"]
         )
         return local_jdata_cpy
-
-    def get_fitting_net(self):
-        """Get the fitting network."""
-        return self.atomic_model.fitting_net
-
-    def get_descriptor(self):
-        """Get the descriptor."""
-        return self.atomic_model.descriptor
```

Comment:
fitting and descriptor access methods may be preserved here. The `dp` model assumes a descriptor and a fitting net as members.